deny.toml

```diff
@@ -36,15 +30,17 @@ ignore = [
     # a `[bans]` entry below. in the prometheus crates, only the protobuf
     # encoder is used, while only the decoder is affected by this advisory.
     "RUSTSEC-2024-0437",
+
+    # The following crates are unmaintained
```

Comment:
Do we have a Jira tick to track this TODO?

Reply:
https://apollographql.atlassian.net/browse/ROUTER-1189